models/repo.go

```diff
@@ -24,7 +25,7 @@ func Init(ctx context.Context) error {
 }
 
 type repoChecker struct {
-	querySQL   func(ctx context.Context) ([]map[string][]byte, error)
+	querySQL   func(ctx context.Context) ([]int64, error)
```

Comment:
The old design `[]map[string]` seems more flexible, why change it?

Reply:
Every SQL is assumed using `select id` for this checker. If we want that flexibility, many places need to be changed. So I don't think it's necessary to keep the flexible if there is no whole design.